### PR TITLE
Docker Compose doesn't use Postgres anymore

### PR DESCRIPTION
**Pull request by cadecairos**
Ensure the docker-compose stack makes use of the Postgres container by default

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ services:
   postgres:
     image: postgres:9.6
     ports:
-      - "5432"
+      - "5432:5432"
```
Comment (cadecairos) [inline docker-compose.yml]: expose the DB to the host, makes it easier to connect with external tools.

File: env.default [hunk 2]
```diff
@@ -15,7 +15,7 @@ ASSET_DOMAIN=network.mofoprod.net
 CONTENT_TYPE_NO_SNIFF=True
 CORS_WHITELIST=*
 CORS_REGEX_WHITELIST=
-DATABASE_URL=
+DATABASE_URL=postgres://postgres@postgres:5432/postgres
```
Comment (cadecairos) [inline env.default]: I tried using environment variables in the docker-compose.yml file but had no luck there.

File: docker-compose.yml (hunk 1 above)
Comment (patjouk) [reply docker-compose.yml]: I would go for that change. @Pomax is it a problem?

File: env.default (hunk 2 above)
Comment (cadecairos) [reply env.default]: What I mean is that in docker-compose I tried setting `POSTGRES_PASSWORD`, `POSTGRES_USER`, and `POSTGRES_DB` to override the default user/pass/db combo on container creation but it never seemed to work right.

**Comment by cadecairos** [comment]
Looks like I skipped running the initial `inv docker-new-env` step and manually copied the default env file before running `inv docker-catchup`. This PR is not needed.

**Pull request closed without merging**